src/components/Text.tsx

```diff
@@ -121,30 +115,30 @@ function Text(props) {
   );
 }
 
-Text.propTypes = {
-  id: PropTypes.string,
-  as: PropTypes.oneOf(AS),
-  ...responsiveMarginType,
-  ...responsivePropType("textStyle", PropTypes.oneOf(TEXT_STYLES)),
-  color: (props) => {
-    allowedColors.forEach(({ textStyles, allowedColors }) => {
-      if (
-        textStyles.includes(props.textStyle) &&
-        !allowedColors.includes(props.color)
-      ) {
-        return new Error(
-          `Text: color="${props.color}" is not allowed for textStyle="${
-            props.textStyle
-          }". Must be one of: ${formatArray(allowedColors)}`
-        );
-      }
-    });
-  },
-  align: PropTypes.oneOf(ALIGNS),
-  wrap: PropTypes.bool,
-  role: PropTypes.string,
-  children: PropTypes.node,
-  testId: PropTypes.string,
-};
+// Text.propTypes = {
+//   id: PropTypes.string,
+//   as: PropTypes.oneOf(AS),
+//   ...responsiveMarginType,
+//   ...responsivePropType("textStyle", PropTypes.oneOf(TEXT_STYLES)),
+//   color: (props) => {
+//     allowedColors.forEach(({ textStyles, allowedColors }) => {
+//       if (
+//         textStyles.includes(props.textStyle) &&
+//         !allowedColors.includes(props.color)
+//       ) {
+//         return new Error(
+//           `Text: color="${props.color}" is not allowed for textStyle="${
+//             props.textStyle
+//           }". Must be one of: ${formatArray(allowedColors)}`
```

Comment:
Would be good to find a TypeScript alternative for this warning.

Reply:
I am keeping this proptypes for future prs to add types to this component